sdl/sdl_wrapper.hpp

```diff
@@ -0,0 +1,103 @@
+#pragma once
```

Comment:
Thanks!!

```suggestion
#pragma once

```

Blank line after the `pragma` please :)